src/defaults.js

```diff
@@ -653,6 +653,17 @@ var defaults = {
     filterable: true,
 
     /**
+     * This is used only by FilterBox cell editor.
+     * One of:
+     * * **`'onCommit'`** - Column filter state not set until keyup === `\r` (return/enter key)
+     * * **`'immediate'`** - Column filter state set on each key press
+     * @default
+     * @type {boolean}
+     * @memberOf module:defaults
+     */
+    filteringMode: 'onCommit',
```

Comment:
My instinctive reaction is that filteringMode sounds very application layer. Our grid really is unawares of any notion of filtering